app/Http/Controllers/BlogsCommentsController.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+use App\Blog;
+use App\Comment;
+
+class BlogsCommentsController extends Controller
+{
+    public function store(Blog $blog)
+    {
+        
+        $validated=request()->validate([
+
+            'name'=>['required'],
+
+            'emailid'=>['required'],
+
```

Comment:
Do not give unnecessary lie breaks. We use line breaks to separate logical code blocks. Here, validate is single logical code block.